documentation/IDTA-01001/modules/ROOT/pages/includes/IDTA-01001_ValueOnly.adoc

```diff
@@ -11,46 +11,329 @@ Plattform Industrie 4.0; Anna Salari, Publik. Agentur für Kommunikation GmbH, d
+====
+
+|SubmodelElementList a| (value) 
+
+if defined: type equal to SubmodelElementList/valueTypeListElement
+
+====
+If the submodel element list is part the Value-Only serialization of its parent element (e.g. in the context of a submodel or submodel element collection) and the list consists of submodel elements of type Operation or Capability the complete list is ignored. 
+
+If the Value-Only serialization of the list is explicitly requested without context a corresponding error message shall be returned. 
+====
+
+|Entity |statements, globalAssetId, specificAssetId
+|BasicEventElement a|observed
+|Capability a| -
+|Operation a| -
+
+====
+There is not Value-Only serialization for an Operation. However, a Value-Only serialization is useful for its arguments when executing the operation (see Value-Only serialization of "OperationVariable")
+====
+
+|RelationshipElement a|first, second
+|AnnotatedRelationshipElement a|first, second, annotations
+
+2+|*DataElements*
+|Property a| (value)
+
+type equal to Property/valueType
+
+====
+valueId is ignored in Value-Only serialization
+====
+|MultilanguageProperty a| (value)
+
+====
+valueId is ignored in Value-Only serialization
+====
+|Range a|min, max
+
+type equal to Range/valueType
+
+====
+In the Value-Only serialization only number ranges are supported
+====
+
+|ReferenceElement a|(value)
+
+|Blob a|value, contentType
+|File a|value, contentType
+
+2+|*Data Types*
+|OperationVariable a| (value)
+
+|MultiLanguageTextType a| The value of this type is identicial to its "Normal"-format serialization: it consists of a list of pairs: a language tag and the value in the corresponding language
+
+[listing]
+....
+<language-tag> ::= 
+	language tag as defined for values of type langString, i.e.
+	in accordance with IETF https://tools.ietf.org/html/rfc5646#page-5[RFC 5646], 
+	the language names match the following regular expression:
+
+		^[a-z]\{2,4}(-[A-Z][a-z]\{3})?(-([A-Z]\{2}|[0-9]\{3}))?$
+	
+....	
+
+	e.g. "en" for Englisch
+|===
 
-* Submodel elements defined in the submodel other than the ones mentioned above are not subject to serialization of that SerializationModifier.
 
-*Data type to value mapping* footnote:[cf. https://openmanufacturingplatform.github.io/sds-documentation/bamm-specification/2.0.0/datatypes.html]
+A submodel element is omitted if no value is defined for it for the requesting submodel or submodel element.
 
-The serialization of submodel element values is described in the following table. The left column “Data Type” shows the data types which can be used for submodel element values. The data types are defined according to the W3C XML Schema (https://www.w3.org/TR/xmlschema-2/#built-in-datatypes and https://www.w3.org/TR/xmlschema-2/#built-in-derived). “Value Range” further explains the possible range of data values for this data type. The right column comprises related examples of the serialization of submodel element values.
+==== How-To Value-Only Serialization
```

Comment:
This definition is better than the previous one but I have the feeling this could still be improved.
Maybe it could make sense to introduce different types of element, e.g.  (rules apply recursively)

- primitive: will be directly serialized as value
   - `Property`
- complex: will be serialized using default serialization
   - `Reference` (actually, I am not sure if this is not correct because the valueOnly serialization of a reference probably does not include e.g. referredSemanticId but this is actually undefined right now I would say)
   - `MultiLanguageProperty`?
   - `Blob`?
   - `File`?
- composite: serialized as JSON object using properties defined in above table in the format "idshort": {value}
   - `Submodel`
   - `SubmodelElementCollection`
   - `Entity`
   - ...
- SubmodelElementList (seems to be a special case): serialized as JSON array 


Das ist definitiv noch nicht ausgereift sondern nur ein erster Gedanke von mir. Wäre interessant zu schauen ob man das zu einer brauchbaren Definition aufbohren kann, dafür habe ich allerdings nicht die Ressourcen.

Reply:
see [comment ](https://github.com/admin-shell-io/aas-specs/pull/375#discussion_r1551038092)above: you require definitions for "complex type" etc. but here you also do not provide definitions, just a (complete) list of elements that needs to be maintained.

Reply:
You are right, the proposed changes are not helping. 
My idea was to make this table shorter by not repeating elements where regular serialization and valueOnly serialization are the same, e.g. Blob, File, Reference(?). Only the types in the category I called `composite` need to be explained further.

The current proposal just adds complexity without real benefit so I am against it. Either keep it as it was previously, i.e. a table of all element types listed explicitly or try to go the way I proposed and find a way to shorten that table by re-using previous definitions (e.g. default serialization for some elements)

Reply:
Sorry, this was your proposal to go. Sounded more easy than it is. If you have something different in mind then please  provide a proposal for the simplified table.

Reply:
My proposal was not to just add subheadings to the table without any kind of explanation what that mean or why they are there. I already provided different categories than you used and also included description/reasoning for each category.